src/main/java/com/hydratereminder/command/total/TotalCommandHandler.java

```diff
@@ -1,15 +1,36 @@
 package com.hydratereminder.command.total;
 
+import com.hydratereminder.HydrateReminderPlugin;
+import com.hydratereminder.chat.ChatMessageSender;
 import javax.inject.Singleton;
 
 import com.hydratereminder.command.CommandHandler;
 
 @Singleton
 public class TotalCommandHandler implements CommandHandler {
 
+    private final ChatMessageSender chatMessageSender;
+
+    private final HydrateReminderPlugin hydrateReminderPlugin;
+
+    public TotalCommandHandler(ChatMessageSender chatMessageSender, HydrateReminderPlugin hydrateReminderPlugin) {
+        this.chatMessageSender = chatMessageSender;
+        this.hydrateReminderPlugin = hydrateReminderPlugin;
+    }
```

Comment:
Injection won't be created. Missing @Inject annotation over constructor.
When trying to run: 
![image](https://user-images.githubusercontent.com/94566117/193451940-73c26883-33b1-4dfd-b1d0-e1f667d70082.png)

Reply:
Added the missing `@Inject` annotation.